scripts/stage-release.sh

```diff
@@ -126,11 +136,14 @@ if [[ "${SKIP_VALIDATION:-false}" != true ]]; then
 fi
 
 echo "Versioning Kroxylicious-junit-extension as ${RELEASE_VERSION}"
+setVersion "${RELEASE_VERSION}"
 
-mvn -q versions:set -DnewVersion="${RELEASE_VERSION}" -DgenerateBackupPoms=false
-echo "Validating things still build"
-mvn -q clean install -Pquick
+#Set the release version in the Changelog
+${SED} -i -e "s_##\sSNAPSHOT_## ${RELEASE_VERSION//./\\.}_g" CHANGELOG.md
```

Comment:
This PR misses adding the corresponding `SNAPSHOT` entry to the changelog.

Reply:
I verified that the CHANGELOG is updated to include `SNAPSHOT`.

```bash
uthor: kwall <kwall@apache.org>
Date:   Thu Mar 14 11:23:48 2024 +0000

    Start next development version

    Signed-off-by: kwall <kwall@apache.org>

diff --git a/CHANGELOG.md b/CHANGELOG.md
index 9001975..604770b 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -4,6 +4,7 @@ Please enumerate all user-facing changes using format `<githib issue/pr number>:

 ## 1.0.0

+## SNAPSHOT
```